src/pytorch_lightning/accelerators/hpu.py

```diff
@@ -39,9 +39,24 @@ def setup_environment(self, root_device: torch.device) -> None:
             raise MisconfigurationException(f"Device should be HPU, got {root_device} instead.")
 
     def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
```

Comment:
Could we add reference to an API doc here?

Reply:
added in the document